libp2p/protocols/pubsub/floodsub.nim

```diff
@@ -118,9 +121,12 @@ method publish*(f: FloodSub,
   trace "publishing on topic", name = topic
   let msg = newMessage(f.peerInfo, data, topic)
   var sent: seq[Future[void]]
+  # start the future but do not wait yet
   for p in f.floodsub[topic]:
     trace "publishing message", name = topic, peer = p, data = data.shortLog
     sent.add(f.peers[p].send(@[RPCMsg(messages: @[msg])]))
+  # wait for all the futures now
+  # TODO FIXME what to do with errors here??
```

Comment:
@dryajov Should we just leave `allFutures` and let it throw here? (repeating the question few times around)

Reply:
same as previous